README.md

```diff
@@ -31,31 +31,34 @@ Please make sure that your environment meets the following requirements:
 
 #### Python Version
 
-The Pantos Client CLI requires **Python 3.10**. Ensure that you have the correct Python version installed before the installation steps. You can download the latest version of Python from the official [Python website](https://www.python.org/downloads/).
+Pantos Common requires **Python 3.12**. Ensure that you have the correct Python version installed before the installation steps. You can download the latest version of Python from the official [Python website](https://www.python.org/downloads/).
```

Comment:
Do we really require 3.12? Or just support it? If we require 3.12, which changes make it incompatible with 3.10?

Reply:
See https://bitpanda.slack.com/archives/C040G3XU9S6/p1714397885229099